lib/serviceDiscovery/engines/consul/index.js

```diff
@@ -125,17 +132,15 @@ class ConsulService extends Service {
 		});
 
 		this._watch.on('error', (error) => {
-			logger.error.data(error).log('error while running discovery process');
+			logger.error.data(error).log('Error while running discovery process');
 		});
-
-		logger.debug('discovery started');
 	}
 
 	/**
 	 * Stop discovery and announcements
 	 */
 	close() {
-		this._announces = [];
+		this._announces.splice(0, this._announces.length);
```

Comment:
If you guys really wanna go there, an easier (and much faster) way to empty out an array:

```js
this._announces.length = 0;
```